driver/main.c

```diff
@@ -1408,6 +1408,12 @@ static inline struct event_data_t *manage_socketcall(struct event_data_t *event_
 {
 	bool is_syscall_return;
 	int return_code = convert_network_syscalls(event_data->event_info.syscall_data.regs, &is_syscall_return);
+	if (return_code == -1)
```

Comment:
below in this function there is a check `if(return_code == PPME_GENERIC_E)` if we decide to change the return value of `convert_network_syscalls` we can remove it

Reply:
Done!